gce_workload_cert_refresh/main.go

```diff
@@ -229,19 +235,21 @@ func refreshCreds() error {
 	if err != nil {
 		return fmt.Errorf("Error getting project ID: %v", err)
 	}
-	domain := fmt.Sprintf("%s.svc.id.goog", project)
-	logger.Infof("Rotating workload credentials for domain %s", domain)
 
 	wisMd, err := getMetadata("instance/workload-identities")
 	if err != nil {
-		return fmt.Errorf("Error getting workload-identities: %v", err)
+		logger.Infof("No workload identities found: %v", err)
```

Comment:
Any special reason to be downgrading this from `Error` to `Info`?

Reply:
yep, because this represents the case where workload identity is not present. not being present is a valid scenario, not an error